tests/test_mixed.py

```diff
@@ -78,6 +78,28 @@ def go():
         for i in range(3):
             self.loop.run_until_complete(go())
 
+    def test_sync_put_async_join(self):
+        q = janus.Queue(loop=self.loop)
+
+        for i in range(5):
+            q.sync_q.put(i)
+
+        @asyncio.coroutine
+        def do_work():
+            yield from asyncio.sleep(1)
```

Comment:
Can you please add `loop=self.loop` here? Looks like this is reason why on 3.4 test stuck.

Reply:
sure, thanks for pointing this out 👍